vulyk/models/stats.py

```diff
@@ -25,6 +29,7 @@ class WorkSession(Document):
     user = ReferenceField(User, reverse_delete_rule=CASCADE, required=True)
     task = ReferenceField(AbstractTask, reverse_delete_rule=CASCADE,
                           required=True)
+    task_type = StringField(max_length=50, required=True, db_field='taskType')
     answer = ReferenceField(AbstractAnswer, reverse_delete_rule=CASCADE)
 
     start_time = DateTimeField(required=True)
```

Comment:
Also I have noticed naming convention is being violated for the whole collection and for both `start_time` and `end_time` fields. Should I possibly rename them?